target/linux/ramips/dts/mt7621_zte_mf286c3.dts

```diff
@@ -0,0 +1,350 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "mt7621.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/leds/common.h>
+
+/ {
+	compatible = "zte,mf286c3", "mediatek,mt7621-soc";
+	model = "ZTE MF286C3";
+
+	chosen {
+		bootargs = "console=ttyS0,57600n8";
+		bootargs-override = "console=ttyS0,57600n8";
+	};
+
+	aliases {
+		led-boot = &led_status;
+		led-failsafe = &led_status;
+		led-running = &led_status;
+		led-upgrade = &led_status;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		pinctrl-names = "default";
+		pinctrl-0 = <&led_pins>;
+
+		led-6  { /* p0_0 sig00 */
+			color = <LED_COLOR_ID_BLUE>;
+			function = LED_FUNCTION_INDICATOR;
+			function-enumerator = <1>;
+			gpios = <&aw9523 0 GPIO_ACTIVE_LOW>;
```

Comment:
Are you sure, that this LED and others on `&aw9523` are not handled by the modem? On ath79 counterparts of the device, they were, although the controller was seemingly accessible by the router part.